### ISSUE TITLE
NZBHydra2 broken after update - 502 error

### ISSUE THREAD

**Issue by bezzpezz** [Cloudbox/Cloudbox#434]
**Describe the bug**

I updated Cloudbox via the ansible tag command and everything works great except for nzbhydra2.
I get a 502 error - It's like someone has changed the port or something in the docker container (installed by Cloudbox) and it hasn't been updated in Cloudbox Nginx ???

**To Reproduce**
1. update Cloudbox using ansible tag
2. try to load / log into nzbhydra2 (working fine pre update) via https://nzbhyrda2.mydomain.com
3. See error 502 bad gateway

**Expected behavior**
browse to https://nzbhyrda2.mydomain.com and have a working web UI

**Screenshots**
![image](https://user-images.githubusercontent.com/6962793/97839965-f02b4200-1d2e-11eb-8e20-61488b6162a1.png)

**Logs**
Link to debug or trace log files..

**System Information**

 - Cloudbox Version: Master (latest) - whatever the latest pull is as of 1/11/2020
 - Operating System: [e.g. Ubuntu Server 18.04 LTS]
 - Linux Kernel:  5.4.0-52-generic

**Additional context**
I am not sure how to update manually update nginx within the container if this is indeed the problem.

**Comment by RXWatcher** [comment]
Hydra2 doesnt use nginx. It's a Java application. You need to see what errors are being produced by the container. Please run ```docker logs -f nzbhydra2``` and post the output.

**Comment by bezzpezz** [comment]
RXWatcher - Thank you for your reply,

Here is the log output:

``[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-start-container: executing...

----------------------------------------------------------------------
ENVIRONMENT
----------------------------------------------------------------------
PUID=1000
PGID=1000
UMASK=2
TZ=Asia/Singapore
ARGS=
DEBUG=no
----------------------------------------------------------------------

Executing usermod...
usermod: no changes
Applying permissions to /config
[cont-init.d] 00-start-container: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.

Does anything seem out of the ordinary?
The Hydra2 docker seems to be running fine and I can log into it's console via Portainer etc.

**Comment by owine** [comment]
Check if your database became corrupted. It happens sometimes and happened to me with the most recent update. Upstream issue

**Comment by bezzpezz** [comment]
OK all fixed now, Thank you guys for your input!

For those who are facing the same problem, here are the steps to Fix:

1. I backed up (renamed) the /opt/nzbhydra2 folder to nzbhydra2_bak
2. Reinstalled nzbhydra2 via anisble tag:- `cd ~/cloudbox` then `sudo ansible-playbook cloudbox.yml --tags nzbhydra2`
3. I then copied the backed up directory (nzbhydra2_bak) to my local PC via SCP command, use "file browser" or something if you have it installed
4. Logged into nzbhydra2 webpage and restored the latest backup zip file: System -> Backup

Everything fixed and working

Cheers!
